src/app/routes/registration/select-asset-dialog/select-asset-dialog.component.html

```diff
@@ -45,7 +45,14 @@ <h4 mat-dialog-title class="mb-0 d-flex justify-content-between align-items-cent
             <td mat-cell *matCellDef="let element">
                 <div class="d-flex justify-content-end">
                     <button mat-icon-button aria-label="Use" (click)="selectAsset(element)">
-                        <mat-icon class="btn-color-primary">check_circle</mat-icon>
+                        <mat-icon
```

Comment:
Why are you adding new line after html tag?

Reply:
As for me, it is easier to read. Corrected to one line.